src/lib/services/invoice-service.js

```diff
@@ -124,7 +123,14 @@ const decorateInvoiceWithCRMData = (invoice, context) => {
     'company'
   )
 
-  return invoice.fromHash(pickBy(properties))
+  return invoice.fromHash(removeFalseyValues(properties))
+}
+
+const removeFalseyValues = (object) => {
+  // Convert the object into an array of key-value pairs, then filter out any entires with falsey values
+  const entries = Object.entries(object).filter(([_, value]) => !!value)
```

Comment:
Does this need to be `!!value` or would `value` work as well?

Reply:
These changes all worked 😄